torch/testing/_internal/inductor_utils.py

```diff
@@ -67,6 +70,17 @@ def test_cpu():
     for gpu in GPU_TYPES
 )
 
+_desired_test_bases = get_desired_device_type_test_bases(allow_xpu=True)
+RUN_GPU = (
+    HAS_GPU
+    and any(is_gpu(getattr(x, "device_type", "")) for x in _desired_test_bases)
+)
+
+RUN_CPU = (
+    HAS_CPU
+    and any(getattr(x, "device_type", "") == "cpu" for x in _desired_test_bases)
+)
+
 def _check_has_dynamic_shape(
```

Comment:
Should we put these two flags in torch/testing/_internal/common_utils.py because they are general flags.

Reply:
Thanks for your review, here the RUN_GPU which need HAS_GPU also check triton availability, this is Inductor specific , so I place them in inductor_utils the same as HAS_GPU.